indico/modules/events/editing/client/js/components/Timeline/judgment/JudgmentBox.jsx

```diff
@@ -0,0 +1,82 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import React, {useState, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {Button, Dropdown} from 'semantic-ui-react';
+
+import {Translate} from 'indico/react/i18n';
+
+import {blockPropTypes} from '../util';
+import AcceptRejectForm from './AcceptRejectForm';
+
+import './JudgmentBox.module.scss';
+
+export default function JudgmentBox({block, onClose, judgmentType: _judgmentType, options}) {
+  const [judgmentType, setJudgmentType] = useState(_judgmentType);
+  const [loading, setLoading] = useState(false);
+  const optionsMap = useMemo(() => _.keyBy(options, 'value'), [options]);
+
+  const option = optionsMap[judgmentType];
```

Comment:
Can't this be simplified to

    const option = options.find(x => x.value === judgmentType);